test/smoke_test.py

```diff
@@ -90,7 +90,9 @@ def main() -> None:
     if torch.cuda.is_available():
         smoke_test_torchvision_decode_jpeg("cuda")
         smoke_test_torchvision_resnet50_classify("cuda")
-        smoke_test_compile()
+
```

Comment:
Please add `TODO: Remove me once https://github.com/pytorch/pytorch/issues/110436 is closed

Reply:
Done!